sources/launcher/Xenko.Launcher/ViewModels/RecentProjectViewModel.cs

```diff
@@ -72,6 +76,33 @@ private void Explore()
             explorer.Start();
         }
 
+        private void Remove()
+        {
+            //Remove files that's was deleted or upgraded by xenko versions <= 3.0
+            if (string.IsNullOrEmpty(this.XenkoVersionName) || string.Compare(this.XenkoVersionName, "3.0", StringComparison.Ordinal) <= 0)
```

Comment:
What's the reason for checking version 3 here? Previous versions can still be installed.

Reply:
This check is not a block for old versions, it is only for executing extra codes. Older packages that have been added / removed / deleted by xenko versions <= 3.0 need to be handled differently.

Reply:
Ok.